test/index.js

```diff
@@ -110,6 +110,15 @@ const checks = [
   [':attr(funding, :attr([type=GitHub i]))'],
   [':attr(arbitrary, foo, :attr(funding, [type=GH]))'],
 
+  // :outdated pseudo
+  [':outdated'],
+  [':outdated(any)'],
```

Comment:
If we're not validating these in `@npmcli/query` what's the point of testing these arbitrary (to this module) strings?

Reply:
i removed all but the first two since those are all that's necessary for test coverage purposes